site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+
+1. To choose other tables, clear the search, put the table name and select the checkbox. Do this for the following tables:
+
+   * sys_audit_delete
+   * task
+
+ > aside positive
+ >   Hint: Clear the search fields, and then select the title **Status** to sort and show all the tables you selected.
+
+ ![Select](assets/Select.png)
+
+
+1. Select **Start Ingestion**. The select windows closes and you get the message "Loading Data" from the main Connector window. In addition to the tables you choose, three system tables will also be loaded. These are necessary to build the views on the raw data: sys_dictionary, sys_db_object, and sys_glide_object.
+
+![load](assets/load.png)
+
+You receive a message indicating success:
+
+![success](assets/success.png)
+
+> aside negative
+> Don't stop the ingest too quickly. Make sure the views are built in the destination database first.
+
+## Connector Monitoring (Query Sync History)
+ Duration: 5
+
+In the connector interface, choose **Query Sync History.** A worksheet
+opens with several SQL queries you can execute to get monitoring
+information. Here are some examples:
+
+```SQL
+// Get general information about all ingestions
+SELECT * FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.connector_stats;
+
+// Search for information about particular table ingestions
+SELECT * FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.connector_stats WHERE table_name = '<table_name>';
+
+// Check connector configuration
+SELECT * FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.global_config;
+
+// Calculate ingested data volume
+WITH d as (
+    SELECT
+        table_name,
+        last_value(totalrows) OVER (PARTITION BY table_name ORDER BY run_end_time) AS row_count
+    FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.connector_stats
+)
+SELECT table_name, max(row_count) as row_count FROM d GROUP BY table_name ORDER BY table_name;
+
+// Connector runtime (minutes from start)
+SELECT timediff('minute', min(run_start_time), max(run_end_time)) AS connector_runtime_in_minutes
+FROM SNOWFLAKE_CONNECTOR_FOR_SERVICENOW.public.connector_stats;
+```
+
+
+## Setting reader role permissions
+Duration: 3
+
+Now that you have ingested some data, let's alter the **servicenow_reader_role** to give it access to the database, schema, future tables, future views, and virtual warehouse.
+```SQL
+USE ROLE accountadmin;
+CREATE ROLE IF NOT EXISTS servicenow_reader_role IF NOT EXISTS;
+GRANT USAGE ON DATABASE SERVICENOW_DEST_DB TO ROLE servicenow_reader_role;
+GRANT USAGE ON SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role; 
+GRANT SELECT ON FUTURE TABLES IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
+GRANT SELECT ON FUTURE VIEWS IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
+GRANT SELECT ON ALL TABLES IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
+GRANT SELECT ON ALL VIEWS IN SCHEMA DEST_SCHEMA TO ROLE servicenow_reader_role;
```

Comment:
```suggestion
GRANT USAGE ON SCHEMA SERVICENOW_DEST_DB.DEST_SCHEMA TO ROLE servicenow_reader_role; 
GRANT SELECT ON FUTURE TABLES IN SCHEMA SERVICENOW_DEST_DB.DEST_SCHEMA TO ROLE servicenow_reader_role;
GRANT SELECT ON FUTURE VIEWS IN SCHEMA SERVICENOW_DEST_DB.DEST_SCHEMA TO ROLE servicenow_reader_role;
GRANT SELECT ON ALL TABLES IN SCHEMA SERVICENOW_DEST_DB.DEST_SCHEMA TO ROLE servicenow_reader_role;
GRANT SELECT ON ALL VIEWS IN SCHEMA SERVICENOW_DEST_DB.DEST_SCHEMA TO ROLE servicenow_reader_role;
```
Or `USE DATABASE SERVICENOW_DEST_DB;` at the beginning, otherwise these might not work.